packages/FileUpload/src/FileList/index.tsx

```diff
@@ -23,6 +26,31 @@ const FileList = ({
           </div>
           <div className="preview-filename-container">
             <p className="preview-filename">{file.name}</p>
+            {file.type === IMAGE && (
```

Comment:
Maybe you can add a variable `const isImage = file.type === IMAGE` & use it in line 29 & also 21 where is repeated

Reply:
I don't think I can create that constant within a map, within a render :(

Reply:
True! I thought that `file` comes from props not from the mapping, my bad!